Fix for building on Solaris.
This should be investigated for sending upstream.

--- a/mozilla/moz.build.orig	2015-04-19 15:26:14.440822480 +0530
+++ b/mozilla/moz.build	2015-04-19 15:26:27.097957297 +0530
@@ -16,7 +16,6 @@ add_tier_dir('base', ['config', 'python'
 if not CONFIG['JS_STANDALONE']:
     CONFIGURE_SUBST_FILES += [
         'mozilla-config.h',
-        'tools/update-packaging/Makefile',
     ]
 
     add_tier_dir('base', ['build', 'probes'])
